app/assets/javascripts/application.js

```diff
@@ -15,3 +15,13 @@
 //= require twitter/bootstrap
 //= require turbolinks
 //= require_tree .
+
+$( document  ).ready(function() {
+  $('#item_delivery_method').change(function(){
+    if ($('#item_delivery_method').val() == 'electronic'){
+      $('.item-form-ticket').show();
+    }else{
+      $('.item-form-ticket').hide();
+    }
+  });
+});
```

Comment:
Javascript!! Extra :star: 's for this! 

so I'm guessing bootstrap gives the delivery method selection field a class of #item_deliery_method. So that when that field is changed, it'll determine wether or not to show the #item-form-ticket field which I see is a file_field in the new ticket view?

Very cool! :star: :star: :star: